sdks/java/core/src/main/java/org/apache/beam/sdk/io/BigQueryIO.java

```diff
@@ -201,8 +201,9 @@
  *
  * <p>See {@link BigQueryIO.Write} for details on how to specify if a write should
  * append to an existing table, replace the table, or verify that the table is
- * empty. Note that the dataset being written to must already exist. Write
- * dispositions are not supported in streaming mode.
+ * empty. Note that the dataset being written to must already exist. In streaming mode, the Write
+ * disposition must be {@link WriteDisposition#WRITE_EMPTY empty} or
```

Comment:
I question whether WRITE_EMPTY ever makes sense when using the the streaming write API. Specifically, unless we're making sure we never attempt to write to a non-empty table, it will get a lot of errors. And it seems like the presence of late data, or additional data arriving, could lead to this problem happening.

@dhalperi @dpmills Any thoughts?